tests/backends/sqlite/tests.py

```diff
@@ -162,3 +162,27 @@ def create_object():
         thread.start()
         thread.join()
         self.assertEqual(Object.objects.count(), 2)
+
+
+@unittest.skipUnless(connection.vendor == 'sqlite', 'SQLite tests')
+class IntrospectionTests(TestCase):
+    def test_get_primary_key_column(self):
+        """
+        Validate that we can get the field name of the primary key.
+        Supports quoted and non-quoted field names.
+        """
+        with connection.cursor() as cursor:
+            cursor.execute(
```

Comment:
Use a single line. Our [style guide](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style) allows lines up to 119 characters if it helps readability. Also, the test should cleanup after itself using `DROP TABLE` in `try/finally`.

Reply:
Understood, I'll make this correction.